pages/citation.md

```diff
@@ -5,29 +5,64 @@ title: Citation
+[https://doi.org/10.5065/a8pp-4358](https://doi.org/10.5065/a8pp-4358), provides a persistent link to the 
+GeoCAT web page, and can be used to cite the GeoCAT project as a whole.
+
+In addition, each GeoCAT tool is assigned a separate DOI per version to allow users to access older 
+releases. This ensures that users are not only able to cite the specific software version through DOIs 
+but are also able to download & use the corresponding release for reproducibility purposes.
 
 If you author a paper that involves data analysis and/or visualization performed with the use of any 
 GeoCAT software, we would like to ask you to please cite GeoCAT software toolkits you used. This helps 
-us better understand the impact of the software on the scientific community, which in turn helps us 
-maintain support for the effort.
+(1) us better understand the impact of the software on the scientific community, which in turn helps us 
+maintain support for the effort, as well as (2) make the research made with scientific software tools 
+reproducible.
+
+
+### Citing the whole GeoCAT effort
+
+You can cite the entire GeoCAT project using the following citation text:
+
+**Visualization & Analysis Systems Technologies. (2019).
+Geoscience Community Analysis Toolkit. Boulder, CO: UCAR/NCAR - Computational 
+and Informational System Lab. doi:10.5065/a8pp-4358.**
+
 
 ### Citing GeoCAT software packages
 
-You can cite any GeoCAT software using the following citation format:
+If you would like to cite any GeoCAT software without referring to a specific version, use
+the following text:
+
+**Visualization & Analysis Systems Technologies. (\<Year\>).
+Geoscience Community Analysis Toolkit: \<Software package name\> [Software].
+Boulder, CO: UCAR/NCAR - Computational and Informational System Lab. doi:\<DOI\>.**
+
+DOI for each GeoCAT software tool can be found in their own GitHub repository or documentation. 
+In the above citation text, update the software package name and the year of package's 
+initialization as appropriate. For example, if you used GeoCAT-comp (i.e. the computational 
+component of GeoCAT):
+
+**Visualization & Analysis Systems Technologies. (2019).
+Geoscience Community Analysis Toolkit: GeoCAT-comp [Software].
+Boulder, CO: UCAR/NCAR - Computational and Informational System Lab. doi:10.5281/zenodo.6607205.**
+
+Instead, if you would like to cite a specific version of any GeoCAT software, use the following 
+text:
 
-**Visualization & Analysis Systems Technologies. (Year). 
-Geoscience Community Analysis Toolkit (\<Software package name\> version \<version_number\>) [Software]. 
-Boulder, CO: UCAR/NCAR - Computational and Informational System Lab. doi:10.5065/A8PP-4358.**
+**Visualization & Analysis Systems Technologies. (Year).
+Geoscience Community Analysis Toolkit: GeoCAT-comp (v\<version\>) [Software].
+Boulder, CO, USA: UCAR/NCAR - Computational and Informational System Lab. doi:\<DOI\>.**
```

Comment:
In line 52, you state that the user can use this format to cite a specific version of *any* GeoCAT software. However, your format specifically uses GeoCAT-comp. Since you provide an example below this, why don't we make this template as general as possible?

```suggestion
**Visualization & Analysis Systems Technologies. (Year).
Geoscience Community Analysis Toolkit: \<Software package name\> (v\<version\>) [Software].
Boulder, CO, USA: UCAR/NCAR - Computational and Informational System Lab. doi:\<DOI\>.**
```

Reply:
Great catch, thanks!